### PR TITLE
Image to speech folder is added

### LINKED ISSUE #706: Image to speech folder
# Aim

- To add a programme which will convert Image to speech

### PR DESCRIPTION
**Pull request by Subhradeep10**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Fixes #706 

Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have created a helpful and easy to understand `README.md`
- [ ] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by kaustubhgupta** [comment]
@Subhradeep10 May I know where is the issue for this PR?

**Comment by Subhradeep10** [comment]
@kaustubhgupta The Mentors are reviewing it..

**Comment by kaustubhgupta** [comment]
@Subhradeep10 I am the mentor of this project and I haven't got any issue from your side

**Comment by kaustubhgupta** [comment]
@Subhradeep10 ?

**Comment by Subhradeep10** [comment]
@kaustubhgupta I did'nt mentioned any issue..I contruibuted my own folder in amazing pything scripts repo

**Comment by Subhradeep10** [comment]
@kaustubhgupta Actually I am new to git and github ...So it will take lil bit time to familiarize it....But I think I didn't raised any issue

**Comment by kaustubhgupta** [comment]
> @kaustubhgupta Actually I am new to git and github ...So it will take lil bit time to familiarize it....But I think I didn't raised any issue

Then raise an issue for this

**Comment by kaustubhgupta** [comment]
@Subhradeep10 ?

**Review by santushtisharma10 — CHANGES_REQUESTED** [review]
@Subhradeep10 add a readme file

File: Image to Speech/image_to_text.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import pytesseract
+from PIL import Image
+from gtts import gTTS
+import os
+
+pytesseract.pytesseract.tesseract_cmd = r'C:\Program Files\Tesseract-OCR\tesseract.exe'
+
+img = Image.open('4.jpg')
+
```
Comment (santushtisharma10) [inline Image to Speech/image_to_text.py:9]: @Subhradeep10  take the file path as input from user instead of hard coding it

**Review by santushtisharma10 — CHANGES_REQUESTED** [review]
@Subhradeep10 also delete main.py file. There is no use of it

File: Image to Speech/image_to_text.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import pytesseract
+from PIL import Image
+from gtts import gTTS
+import os
+
+pytesseract.pytesseract.tesseract_cmd = r'C:\Program Files\Tesseract-OCR\tesseract.exe'
```
Comment (kaustubhgupta) [inline Image to Speech/image_to_text.py:6]: take the input for this also

File: Image to Speech/image_to_text.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+print(value)
+
+mytext = value
+language = 'en'
+
+output = gTTS(text=mytext, lang=language, slow=True)
+output.save('output.mp3')
+
+os.system("start output.mp3")
```
Comment (kaustubhgupta) [inline Image to Speech/image_to_text.py:20]: After playing this audio, delete the file created (output.mp3). Suggested module: os

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Delete the output.mp3 file from this repo. Other changes:

**Comment by kaustubhgupta** [comment]
@Subhradeep10 Have you looked at the suggested changes?

**Comment by kaustubhgupta** [comment]
@Subhradeep10 Will you fix this PR?

**Comment by kaustubhgupta** [comment]
PR closed due to no response from participant

**Pull request closed without merging**